consensus/istanbul/backend/backend.go

```diff
@@ -251,6 +255,27 @@ func (sb *Backend) Commit(proposal istanbul.Proposal, bitmap *big.Int, seals []b
 	if err != nil {
 		return err
 	}
+
+	parent := sb.chain.GetHeader(h.ParentHash, h.Number.Uint64()-1)
+	// If the parent was not the genesis block then try to use the parent's
+	// extras as the seal for this block
+	if parent != nil {
+		parentExtras, err := types.ExtractIstanbulExtra(parent)
```

Comment:
If we do things this way, I believe we will only ever write 2F+1 signatures to the `parentSeal`, even if we received more signatures than that